### DISCUSSION TITLE
Extending functinality of spack for benchmarking installed application with sample dataset

### DISCUSSION THREAD

**Discussion by samcom12** [spack/spack#22042] [General]
Hi... Can we think of benchmarking installed application with sample dataset? May be in future.

**Comment by tgamblin** [comment]
I think it's a good idea. At the moment there is a lot of work around builds scheduled, and I'm not sure we'll be able to get to this for some time.  [ReFrame](https://github.com/eth-cscs/reframe) and [Pavilion2](https://github.com/hpc/pavilion2) might be worth a look for designing scaling studies / sets of runs around a Spack package.
